internal/help/help.go

```diff
@@ -0,0 +1,85 @@
+package help
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/nestoca/joy/internal/config"
+)
+
+const allCommandsKey = "_all"
+
+func WrapError(cmd *cobra.Command, cmdError error) error {
+	if cmdError == nil {
+		return nil
+	}
+
+	message, err := getMessageFromCommand(cmd, cmdError)
+	if err != nil {
+		return err
+	}
+	if message == "" {
+		return cmdError
+	}
+
+	return fmt.Errorf("%w\n\n%s", cmdError, message)
+}
+
+func AugmentCommandHelp(cmd *cobra.Command) {
+	baseHelpFunc := cmd.HelpFunc()
+	cmd.SetHelpFunc(func(cmd *cobra.Command, args []string) {
+		baseHelpFunc(cmd, args)
+
+		message, err := getMessageFromCommand(cmd, nil)
+		if err != nil {
+			panic(err)
+		}
+		if message != "" {
+			fmt.Println("\n" + message)
```

Comment:
I suspect the message should go to stderr! Pretty sure help text goes there by default, and so the text is now interleaved between two file descriptors. Though correctly formatted in the terminal for sure :)